webpack/assets/javascripts/react_app/components/hosts/storage/vmware/controller/disk/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import Select from '../../../../../common/forms/Select';
+import Checkbox from '../../../../../common/forms/Checkbox';
+import TextInput from '../../../../../common/forms/TextInput';
+import './disk.scss';
+
+const Disk = (
+  {
+    removeDisk,
+    updateDisk,
+    name,
+    storagePod,
+    datastores,
+    storagePods,
+    dataStore,
+    size,
+    thinProvision,
+    eagerZero
+  }
+) => {
+  const selectableStores = datastores.map(attribute => {
+    const key = Object.keys(attribute)[0];
+    const value = Object.values(attribute)[0];
+
+    return <option key={key} value={key}>{value}</option>;
+  });
+
+  const selectablePods = storagePods.map(attribute => {
+    const key = Object.keys(attribute)[0];
+    const value = Object.values(attribute)[0];
+
+    return <option key={key} value={key}>{value}</option>;
+  });
+
+  return (
+    <div className="disk-container">
+      <TextInput
```

Comment:
currently they will all show "hard disk" since it is taken from consts. should i implement an ajax request to fetch the new volume name from the server?